content/en/integrations/guide/azure-architecture-and-configuration.md

```diff
@@ -0,0 +1,220 @@
+
+**Note**: You can enable resource logs without any filters to send all resource logs, as shown below.
+
+{{< img src="integrations/guide/azure_architecture_and_configuration/datadog_agent_build_metrics_and_logs.png" alt="Diagram of the Datadog Agent build" >}}
+
+Diagnostic settings created by the Datadog resource include all log categories, are configured with `Send to Partner Solution`, and send logs back to the originating Datadog resource. They follow the naming format `DATADOG_DS_V2_<UNIQUE_IDENTIFIER>`.
+
+Any manual changes to the resource, including deleting it, are reverted within a few minutes.
+
+See below an example of a Diagnostic Setting created by a Datadog resource:
+
+{{< img src="integrations/guide/azure_architecture_and_configuration/diagnostic_setting.png" alt="Diagram of the diagnostic setting" >}}
+
+### Alternate Configuration Options for Log Forwarding with the Azure Native Integration
+
+The one-click buttons to enable logs in the Datadog resource automate the process of adding diagnostic settings. In some cases, organizations may want to manage and configure diagnostic settings themselves, while still taking advantage of the automated log forwarding capability with the Azure Native integration. 
+
+Manually created diagnostic settings are not impacted by log settings on the Datadog resource, and are not deleted based on the tag rules specified in the Datadog resource. Resource logs do not have to be enabled on the Datadog resource in order for manual log forwarding to work. However, the Datadog resource being used for log forwarding must not be in a disabled state.
+
+Reasons for manually managing Diagnostic Settings include:
+
+  1. Infrastructure-as-code Policies
+       Strict internal policies around IaC that require all resources to to be created and managed deterministically (for example, if automatic creation of diagnostic settings by the Datadog resource would cause an unresolvable conflict between the desired and the actual state).
+
+  2. Limiting Resource Log Categories
+       As the diagnostic settings created automatically by the Datadog resource include all log categories, specifying a subset of these categories requires you to create diagnostic settings yourself.  
+       **Note**: You can also use [exclusion filters][1] to exclude these logs from being indexed upon ingestion into Datadog.
+
+  3. Cross-subscription log forwarding
+       This can be useful if you only want to send logs and no other data from a specific subscription. The Datadog resource used for log forwarding still collects metrics and data from its own subscription. You must register the Resource Provider `Microsoft.Datadog` in each subscription intended to send logs before creating the diagnostic settings.
```

Comment:
```suggestion
       This can be useful if you only want to send logs and no other data from a specific subscription. The Datadog resource used for log forwarding still collects metrics and data from its own subscription. You must register the resource provider `Microsoft.Datadog` in each subscription intended to send logs before creating the diagnostic settings.
```